server.R

```diff
@@ -41,6 +41,39 @@ server <- function(input, output,session) {
       mutation_choices <- dbGetQuery(database, query)
       updateSelectizeInput(session, "mutation_status", choices = mutation_choices$Gene, selected = NULL, server = TRUE)
     }
+
+    # render median-line checkbox
+    if(input$dataset != "" && input$subtype != "Multiplot"){
+      output$toggle_median_ui <- renderUI({
+        checkboxInput(inputId = "toggle_median",
+                      label = "Show Median Line",
+                      value = FALSE)
+      })
+    }
+    # render raw-value checkbox
```

Comment:
Upon close review, I think this would better be expressed as "Log2 scale y-axis" and then have it be checked by default, with the ability to uncheck it.

Reply:
Do you want to the median line to be on by default as well?

Reply:
Yes, I think it's reasonable to be on by default